<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Communicate</class>
 <widget class="QWidget" name="Communicate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>576</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; text-decoration: underline;&quot;&gt;这里是通讯&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toHome">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>回主页</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>901</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameInput">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>551</width>
        <height>231</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="loginButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="toRegister">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>110</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>去注册</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="registerButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordInput_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="hideandshow_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>92</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>显示密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>69</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
       <widget class="QPushButton" name="toLogin">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>120</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>去登录</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="hideandshow">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>显示密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTextEdit" name="chatDisplay">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>751</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 247, 216);
font-size:18px
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="messageInput">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>561</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>410</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>登出</string>
      </property>
     </widget>
     <widget class="QLabel" name="peopleNum">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>当前在线人数：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
